<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShWatchDock</class>
 <widget class="QDockWidget" name="ShWatchDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../glslDevil.qrc">
    <normaloff>:/icons/icons/face-devil-grin_32.png</normaloff>:/icons/icons/face-devil-grin_32.png</iconset>
  </property>
  <property name="windowTitle">
   <string>Watching</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="WatchWindow">
         <property name="toolTip">
          <string>view vector</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/watch-new_32.png</normaloff>:/icons/icons/watch-new_32.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="WatchWindowAdd">
         <property name="toolTip">
          <string>add to active window</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/watch-add_32.png</normaloff>:/icons/icons/watch-add_32.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="WatchDelete">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/watch-delete_32.png</normaloff>:/icons/icons/watch-delete_32.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="SelectionPos">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="ShWatchedTreeView" name="tvWatchList">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShWatchedTreeView</class>
   <extends>QTreeView</extends>
   <header>docks/shproxytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../glslDevil.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>WatchDelete</sender>
   <signal>clicked()</signal>
   <receiver>ShWatchDock</receiver>
   <slot>removeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WatchWindow</sender>
   <signal>clicked()</signal>
   <receiver>ShWatchDock</receiver>
   <slot>newWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WatchWindowAdd</sender>
   <signal>clicked()</signal>
   <receiver>ShWatchDock</receiver>
   <slot>extendWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeSelected()</slot>
  <slot>newWindow()</slot>
  <slot>extendWindow()</slot>
 </slots>
</ui>
